packages/preview-middleware/src/base/config.ts

```diff
@@ -0,0 +1,396 @@
+import type { Logger } from '@sap-ux/logger';
+import { ToolsLogger } from '@sap-ux/logger';
+import type { App, FlpConfig, Intent, InternalTestConfig, MiddlewareConfig } from '../types';
+import { render } from 'ejs';
+import { join, posix } from 'path';
+import { createProjectAccess, getWebappPath, type Manifest, type UI5FlexLayer } from '@sap-ux/project-access';
+import { readFileSync } from 'fs';
+import { mergeTestConfigDefaults } from './test';
+import { type Editor, create } from 'mem-fs-editor';
+import { create as createStorage } from 'mem-fs';
+
+export interface CustomConnector {
+    applyConnector: string;
+    writeConnector: string;
+    custom: boolean;
+}
+
+export interface FlexConnector {
+    connector: string;
+    layers: string[];
+    url?: string;
+}
+
+/**
+ * Internal structure used to fill the sandbox.html template
+ */
+export interface TemplateConfig {
+    basePath: string;
+    apps: Record<
+        string,
+        {
+            title: string;
+            description: string;
+            additionalInformation: string;
+            applicationType: 'URL';
+            url: string;
+            applicationDependencies?: { manifest: boolean };
+        }
+    >;
+    ui5: {
+        libs: string;
+        theme: string;
+        flex: (CustomConnector | FlexConnector)[];
+        bootstrapOptions: string;
+        resources: Record<string, string>;
+    };
+    init?: string;
+    flex?: {
+        [key: string]: unknown;
+        layer: UI5FlexLayer;
+        developerMode: boolean;
+        pluginScript?: string;
+    };
+    locateReuseLibsScript?: boolean;
+}
+
+/**
+ * Static settings
+ */
+export const PREVIEW_URL = {
+    client: {
+        url: '/preview/client',
+        local: join(__dirname, '../../dist/client'),
+        ns: 'open.ux.preview.client'
+    },
+    api: '/preview/api'
+} as const;
+
+/**
+ * Default theme
+ */
+export const DEFAULT_THEME = 'sap_horizon';
+
+/**
+ * Default path for mounting the local FLP.
+ */
+export const DEFAULT_PATH = '/test/flp.html';
```

Comment:
'/test/flpSandbox.html'?

Reply:
Nope, that is the default for the preview-middleware, however, the `fiori-tools-preview` is setting `/test/flpSandbox.html` as default when embedding the middleware for backwards compatibility.